docs/source/install/deb.rst

```diff
@@ -23,7 +23,7 @@ The easiest way to install these is to use the requirements.txt file from the |s
 
 ::
 
-    curl -q -k -O https://downloads.stackstorm.net/releases/st2/0.10dev/requirements.txt
+    curl -q -k -O https://raw.githubusercontent.com/StackStorm/st2/v0.12.1/requirements.txt
```

Comment:
This should actually point to v0.12 branch.  The v0.12.1 is used for staging changes for the patch release that has not yet been released.

```curl -q -k -O https://raw.githubusercontent.com/StackStorm/st2/v0.12/requirements.txt```